src/main/java/com/aws/greengrass/device/DeviceAuthClient.java

```diff
@@ -216,6 +223,10 @@ public boolean canDevicePerform(AuthorizationRequest request) throws Authorizati
             throw new InvalidSessionException(
                     String.format("Invalid session ID (%s)", request.getSessionId()));
         }
+        // Allow all operations from internal components
```

Comment:
Can you add a comment that the workaround above still needs to be there since Moquette isn't using the IPC interface?

Reply:
right.